index.html

```diff
@@ -37,8 +37,128 @@ <h1>I'm Sohail Glad to see you!</h1>
                 </li>
             </ul>
         </div>
-
-    </main>
-    
+<!-- ------------------------------------------------works--------------------------------         -->
+        <div class="shadow-back">
+            <div class="works">
+                <img src="./images/snapshoot-portfolio.png" alt="">
+                <div class="card-work work">
+                    <h3>Tonic</h3>
+                    <ul class="proj">
+                        <li><a href="#">CANOPY</a></li>
+                        <li><div class="dot"></div></li>
+                        <li>Back End Dev</li>
+                        <li><div class="dot"></div></li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="pro">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="language">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button><a href="#">See Project</a></button>
```

Comment:
- [ ] Since you already have an acnhor tag `<a>` there is no nedd to wrapp it inside a button, let's remove the button opening and closing tags :recycle: